maps/RandomZLevels/assistantChamber.dmm

```diff
@@ -21,13 +21,12 @@
 "u" = (/obj/item/weapon/gun/energy/laser/captain,/turf/simulated/floor{icon_state = "white"},/area/awaymission)
 "v" = (/obj/item/weapon/gun/projectile/deagle/gold,/turf/simulated/floor{icon_state = "white"},/area/awaymission)
 "w" = (/obj/item/weapon/sord,/turf/simulated/floor{icon_state = "white"},/area/awaymission)
-"x" = (/obj/item/weapon/gun/energy/lasercannon,/turf/simulated/floor{icon_state = "white"},/area/awaymission)
```

Comment:
The laser cannon seems to be gone entirely rather than replaced with the new path

Reply:
Fiiiiiiine